refresh_test.py

```diff
@@ -145,3 +145,30 @@ def test_refresh_node(self):
         self.end = self.get_current_timestamp()
 
         self.check_timeout()
+
+    def test_node_recovery(self):
+
+        self.log.info('DB cluster is: %s', self.db_cluster)
+        assert len(self.db_cluster.nodes) == 2, "The test requires 2 DB nodes!"
+        stress_queue = self.run_stress_thread(stress_cmd=self.params.get('stress_cmd'))
+        self.get_stress_results(queue=stress_queue)
+        self.verify_stress_thread(cs_thread_pool=stress_queue)
+
```

Comment:
It'll be good to run the stress read here as well, to make sure that before the stop, all keys are there.